README.md

```diff
@@ -332,6 +343,18 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [VR Space Builder](http://vr.greenbushlabs.com/) - Simple VR builder for kids using a form to import models and images.
 - [WebVR API Simulation Extension](https://chrome.google.com/webstore/detail/webvr-api-emulation/gbdnpaebafagioggnhkacnaaahpiefil) - Run WebVR 1.0 content without a supported headset or even a compatible browser/platform. Great for development.
 - [WebVR Solar System](http://vrspace.jmvisualcreativity.es/) - Solar System using A-Frame.
+- [Track](https://samsunginternet.github.io/a-frame-demos/racer.html) - Zooming on the track with the ocean water reflecting the sunset sky. [[Code](https://github.com/SamsungInternet/a-frame-demos/blob/gh-pages/racer.md)]
```

Comment:
can you fix this to point to https://samsunginternet.github.io/a-frame-demos/racer/ and https://github.com/SamsungInternet/a-frame-demos/blob/gh-pages/racer/index.html, respectively?